roles/satellite-clone/tasks/main.yml

```diff
@@ -1,17 +1,22 @@
 ---
-- include_vars: main.yml
-
 # include pre_install_check
 - include: pre_install_check.yml
   when: run_pre_install_check
 
+- name: "check that variables are updated"
```

Comment:
In the case, can you update the task name to something like: `Mandatory check to verify if the required variables are updated` so it makes more sense to developers editing this in future.